docs/source/futures.rst

```diff
@@ -872,6 +872,12 @@ All operations that require talking to the remote worker are awaitable:
 
        n = await counter.n  # attribute access also must be awaited
 
+Generally, all I/O operations should be done using the :code:`compute=False` 
+parameter to avoid asynchronous blocking:
+
+.. code-block:: python
+
+   df3 = await client.compute(dd.read_parquet('/tmp/some.parquet', compute=False))
```

Comment:
[`dd.read_parquet`](https://docs.dask.org/en/stable/generated/dask.dataframe.read_parquet.html) doesn't have a `compute` option. I think this example was supposed to be for [`dd.to_parquet`](https://docs.dask.org/en/stable/generated/dask.dataframe.to_parquet.html) by looking at the discussions in https://github.com/dask/dask/issues/6971

```suggestion
   df3 = await client.compute(dd.to_parquet('/tmp/some.parquet', compute=False))
```